```
stock_scanner/models/scanner_hardware.py:765:26: E231 missing whitespace after ','
stock_scanner/models/scanner_hardware.py:765:35: E231 missing whitespace after ','
stock_scanner/models/scanner_hardware.py:766:25: E231 missing whitespace after ','
stock_scanner/models/scanner_hardware.py:766:34: E231 missing whitespace after ','
stock_scanner/models/scanner_scenario.py:79:21: E251 unexpected spaces around keyword / parameter equals
```

--- a/stock_scanner/models/scanner_hardware.py
+++ b/stock_scanner/models/scanner_hardware.py
@@ -754,7 +754,6 @@ class ScannerHardware(models.Model):
         """
         Retrieve the scenario list for this warehouse
         """
-
         scanner_scenario_obj = self.env['scanner.scenario']
         scanner_scenario_ids = scanner_scenario_obj.search(
             ['|',
@@ -762,8 +761,8 @@ class ScannerHardware(models.Model):
              ('warehouse_ids', 'in', [self.warehouse_id.id]),
              ('parent_id', '=', parent_id),
              '|',
-             ('child_ids','not in',[]),
-             ('step_ids','not in',[])
+             ('child_ids', 'not in', []),
+             ('step_ids', 'not in', [])
              ])
         return scanner_scenario_ids.mapped('name')
 

--- a/stock_scanner/models/scanner_scenario.py
+++ b/stock_scanner/models/scanner_scenario.py
@@ -76,7 +76,7 @@ class ScannerScenario(models.Model):
         ondelete='restrict',
         help='Parent scenario, used to create menus.')
     child_ids = fields.One2many(
-        comodel_name ='scanner.scenario',
+        comodel_name='scanner.scenario',
         inverse_name='parent_id',
         string='Subordinates')
     type = fields.Selection(

